worldengine/astar.py

```diff
@@ -158,11 +158,10 @@ def __init__(self, map_data, width, height):
     def get_node(self, location):
         x = location.x
         y = location.y
-        if x < 0 or x >= self.w or y < 0 or y >= self.h:
-            return None
-        d = self.m[(y * self.w) + x]
-
-        return Node(location, d, ((y * self.w) + x))
+        if (0 <= x < self.w) and (0 <= y < self.h):
```

Comment:
I often read (never checked, though) that (a < x < b) is a lot faster than (a < x and x < b).